### PR TITLE
Add some details to the 2024.2.0 release notes

### PR DESCRIPTION
**Pull request by jesserockz**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [ ] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [x] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by ssieb — DISMISSED** [review]

**Review by kbx81 — DISMISSED** [review]

File: changelog/2024.2.0.rst [hunk 1]
```diff
@@ -18,6 +18,29 @@ ESPHome 2024.2.0 - 21st February 2024
...
+See the :doc:`component documentation </components/micro_wake_word>` for details on how to use it.
+
+Check out the live stream replay below for more details about how it works and came to be.
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/NQIv3nsu7dE"
+            title="YouTube video player" frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen>
+    </iframe>
```
Comment (kbx81) [inline changelog/2024.2.0.rst:35]: I'm unclear on what the linter is complaining about here; we have multiple other instances of this exact code and it's fine.

File: changelog/2024.2.0.rst [hunk 2]
```diff
@@ -18,6 +18,29 @@ ESPHome 2024.2.0 - 21st February 2024
...
+
+
+BME280 changes
+--------------
+
+:ghuser:`apbodrov` has added support for the BME280 sensor over SPI. This change is a breaking change for any existing configurations using
+the BME280 over I2C as the platform name has been changed from ``bme280`` to ``bme280_i2c``. There are no other configuration changes required
+to continue using the BME280 over I2C. See the :doc:`component documentation </components/sensor/bme280>` for the latest configuration variables.
+
```
Comment (ssieb) [inline changelog/2024.2.0.rst:44]:
```suggestion


CSE7766 changes
--------------

:ghuser:`ssieb` has changed the CSE7766 sensor from appearing to be a polling component.
It has a high update rate, so if you don't want to send a lot of data to HA, you can
add the `throttle_average https://esphome.io/components/sensor/#throttle-average`__ filter
with the time period you were using for the update interval (the default is 60s) to each
sensor.  The example in the sensor doc page has been updated to show this.

```

File: changelog/2024.2.0.rst [hunk 3]
```diff
@@ -18,6 +18,39 @@ ESPHome 2024.2.0 - 21st February 2024
...
     Combination, components/sensor/combination, function.svg, dark-invert
 
+micro Wake Word
+---------------
+
+ESPHome implements an on-device wake word detection framework from `microWakeWord <https://github.com/kahrendt/microWakeWord>`__.
+See the :doc:`component documentation </components/micro_wake_word>` for details on how to use it.
+
+Check out the live stream replay below for more details about how it works and came to be.
+.. raw:: html
```
Comment (ssieb) [inline changelog/2024.2.0.rst:29]:
```suggestion

.. raw:: html
```

File: changelog/2024.2.0.rst [hunk 4]
```diff
@@ -18,6 +18,40 @@ ESPHome 2024.2.0 - 21st February 2024
...
+the BME280 over I2C as the platform name has been changed from ``bme280`` to ``bme280_i2c``. There are no other configuration changes required
+to continue using the BME280 over I2C. See the :doc:`component documentation </components/sensor/bme280>` for the latest configuration variables.
+
+
+CSE7766 changes
+--------------
+
+:ghuser:`ssieb` has changed the CSE7766 sensor from appearing to be a polling component.
+It has a high update rate, so if you don't want to send a lot of data to HA, you can
+add the `throttle_average https://esphome.io/components/sensor/#throttle-average`__ filter
```
Comment (ssieb) [inline changelog/2024.2.0.rst]:
```suggestion
add the `throttle_average /components/sensor/#throttle-average`__ filter
```

File: changelog/2024.2.0.rst [hunk 5]
```diff
@@ -18,6 +18,40 @@ ESPHome 2024.2.0 - 21st February 2024
...
+BME280 changes
+--------------
+
+:ghuser:`apbodrov` has added support for the BME280 sensor over SPI. This change is a breaking change for any existing configurations using
+the BME280 over I2C as the platform name has been changed from ``bme280`` to ``bme280_i2c``. There are no other configuration changes required
+to continue using the BME280 over I2C. See the :doc:`component documentation </components/sensor/bme280>` for the latest configuration variables.
+
+
+CSE7766 changes
+--------------
```
Comment (ssieb) [inline changelog/2024.2.0.rst]:
```suggestion
---------------
```

File: changelog/2024.2.0.rst [hunk 6]
```diff
@@ -18,6 +18,40 @@ ESPHome 2024.2.0 - 21st February 2024
...
+the BME280 over I2C as the platform name has been changed from ``bme280`` to ``bme280_i2c``. There are no other configuration changes required
+to continue using the BME280 over I2C. See the :doc:`component documentation </components/sensor/bme280>` for the latest configuration variables.
+
+
+CSE7766 changes
+---------------
+
+:ghuser:`ssieb` has changed the CSE7766 sensor from appearing to be a polling component.
+It has a high update rate, so if you don't want to send a lot of data to HA, you can
+add the `throttle_average /components/sensor/#throttle-average`__ filter
```
Comment (ssieb) [inline changelog/2024.2.0.rst]:
```suggestion
add the `throttle_average </components/sensor/#throttle-average>` filter
```

File: changelog/2024.2.0.rst [hunk 7]
```diff
@@ -18,6 +18,40 @@ ESPHome 2024.2.0 - 21st February 2024
...
+the BME280 over I2C as the platform name has been changed from ``bme280`` to ``bme280_i2c``. There are no other configuration changes required
+to continue using the BME280 over I2C. See the :doc:`component documentation </components/sensor/bme280>` for the latest configuration variables.
+
+
+CSE7766 changes
+---------------
+
+:ghuser:`ssieb` has changed the CSE7766 sensor from appearing to be a polling component.
+It has a high update rate, so if you don't want to send a lot of data to HA, you can
+add the `throttle_average </components/sensor/#throttle-average>` filter
```
Comment (ssieb) [inline changelog/2024.2.0.rst]:
```suggestion
add the `throttle_average <_sensor-filters>` filter
```

File: changelog/2024.2.0.rst [hunk 8]
```diff
@@ -18,6 +18,40 @@ ESPHome 2024.2.0 - 21st February 2024
...
+the BME280 over I2C as the platform name has been changed from ``bme280`` to ``bme280_i2c``. There are no other configuration changes required
+to continue using the BME280 over I2C. See the :doc:`component documentation </components/sensor/bme280>` for the latest configuration variables.
+
+
+CSE7766 changes
+---------------
+
+:ghuser:`ssieb` has changed the CSE7766 sensor from appearing to be a polling component.
+It has a high update rate, so if you don't want to send a lot of data to HA, you can
+add the `throttle_average <_sensor-filters>` filter
```
Comment (ssieb) [inline changelog/2024.2.0.rst]:
```suggestion
add the :ref:`throttle_average <_sensor-filters>` filter
```

File: changelog/2024.2.0.rst [hunk 9]
```diff
@@ -18,6 +18,40 @@ ESPHome 2024.2.0 - 21st February 2024
...
+the BME280 over I2C as the platform name has been changed from ``bme280`` to ``bme280_i2c``. There are no other configuration changes required
+to continue using the BME280 over I2C. See the :doc:`component documentation </components/sensor/bme280>` for the latest configuration variables.
+
+
+CSE7766 changes
+---------------
+
+:ghuser:`ssieb` has changed the CSE7766 sensor from appearing to be a polling component.
+It has a high update rate, so if you don't want to send a lot of data to HA, you can
+add the :ref:`throttle_average <_sensor-filters>` filter
```
Comment (ssieb) [inline changelog/2024.2.0.rst]:
```suggestion
add the :ref:`throttle_average <sensor-filters>` filter
```

**Review by ssieb — APPROVED** [review]

**Pull request merged**